### DISCUSSION TITLE
AI problems

### DISCUSSION THREAD

**Discussion by Philipp-00** [Vdauphin/HeartsAndMinds#1658] [Question and answer]
The AI, both civilian and enemy units, spawns in the triggered area and receives its waypoints. However, it does not move. It remains inactive until it is first shot at. Another odd behavior is that while the AI shoots and hits its target, it keeps aiming its weapon at the ground until it is shot at for the first time. Are there any solutions or compatibility issues, for example with WBK animations? Thank you very much for your help.

[Modpack](https://we.tl/t-fupoyweH1r)

**Comment by Vdauphin** [comment]
Hello,

That sound like an A3 bug? 
H&M does handle the movement of AI or how they aim
You may found help in A3 Official Discord

Cheers

**Reply by Philipp-00** [reply]
Thank you for the answer, I'll see if I get an answer in Discord.

**Comment by Philipp-00** [comment]
As expected, there was no response in the ArmA Discord. However, we found out that the AI ​​is in an ACE animation when it spawns. If I stop it manually it works. Do you perhaps have an idea for this?

**Reply by Philipp-00** [reply]
Add: Civilians are also affected by the animation.
![20241016192119_1](https://github.com/user-attachments/assets/6e6c52f8-bc2e-41a8-a121-cb82cfa701b5)

**Reply by Philipp-00** [reply]
Edit2: We suspect that something is wrong between the Spawn script and the Mod: ACE 3 Extension (Animations and Actions). [Link: https://steamcommunity.com/sharedfiles/filedetails/?id=766491311].
Do you have any idea why this could be and whether it is possible to fix the error other than removing the mod.

**Reply by Vdauphin** [reply]
I don't know what is ACE Animation
There is no magic, H&M spawn AI with `createUnit`
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/delay/createUnit.sqf#L43
You need to ask ACE 3 Extension (Animations and Actions) to fix the issue then
